scripts/.clang-format-version

```diff
@@ -1 +1 @@
-19.1.6
+19.1.7
```

Comment:
`l`: maybe move this and the swiftlint-version to its own PR

Reply:
I will open automerging PRs for this in the future